packages/ocamlformat/ocamlformat.0.3/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/ocaml-ppx/ocamlformat/archive/0.3.tar.gz"
```

Comment:
Something odd happening with the release archive and `.git` being looked for...

```
#=== ERROR while installing ocamlformat.0.3 ===================================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p ocamlformat -j 4
# path         /home/travis/.opam/4.06.0/build/ocamlformat.0.3
# compiler     4.06.0
# exit-code    1
# env-file     /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.env
# stdout-file  /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.out
# stderr-file  /home/travis/.opam/4.06.0/build/ocamlformat.0.3/ocamlformat-13654-f415d0.err
### stderr ###
# File unavailable: .git/HEAD
```

Reply:
Sorry for the noise, I missed a step in the release procedure.